### ISSUE TITLE
Tutorial 1 training still get nan loss

### ISSUE THREAD

**Issue by xiaogangzhu** [kochbj/Deep-Learning-for-Causal-Inference#8]
Hi,
for short tutorial 1 I still get training loss == nan
![Screenshot from 2023-08-08 13-45-50](https://github.com/kochbj/Deep-Learning-for-Causal-Inference/assets/65101307/d4dbfdb1-3996-4941-a15e-3061df1ff788)

**Comment by xiaogangzhu** [comment]
And also Tutorial 3: Semi-parametric extensions to TARNet get nan loss. 

Why get nan loss? Is the tensorflow version or GPU problem?

**Comment by kochbj** [comment]
Thanks for letting me know. I'll look into it as soon as I get the chance!

Best,
Bernie

**Comment by kochbj** [comment]
Hi Xiaogang,

I just tried the short tutorial 1 again and get no issues, using either CPU or the T4 GPU on collab. In general, using GPUs is probably slower than using CPU for these small networks anyways. Are you not running it on collab?

Best,
Bernie

**Comment by xiaogangzhu** [comment]
I tried different version of tensorflow and find that tensorflow>2.10 will get the error. I use tensorflow=2.10.0 and it is fine now!

**Comment by kochbj** [comment]
Glad to hear it! In general, always a good idea to start with recommended package versions.

Warmly,
Bernie
